### PR TITLE
feat: serialise an AggregateError's sub-errors

### PR DESCRIPTION
**Pull request by apaleslimghost**

### CODE REVIEW

**Review by rowanmanning — COMMENTED** [review]
Ooh I didn't know about `AggregateError`! Nice 🙂 happy with the code but not sure why ESLint is failing.

It'd be good to have a line in the README for this too, because [we do for all other properties](https://github.com/Financial-Times/dotcom-reliability-kit/tree/main/packages/serialize-error#serializederror-type) and it'd be good to highlight that this property can be an array or `undefined`.

**Comment by apaleslimghost** [comment]
ah, `AggregateError` is an ES2021 feature, and you're targeting ES2020. it's supported in Node since v15, can we bump your target to ES2021?

File: packages/serialize-error/README.md [hunk 1]
```diff
@@ -100,6 +101,9 @@ This array contains a list of [system codes](https://biz-ops.in.ft.com/list/Syst
 
 This is an error instance extracted from the `error.cause` property, which is serialized before being assigned. It defaults to `null`.
 
+#### `SerializedError.cause`
```
Comment (rowanmanning) [inline packages/serialize-error/README.md:104]: Small copy/paste error

```suggestion
#### `SerializedError.errors`
```

**Review by rowanmanning — APPROVED** [review]
Nice, thanks for doing this!

**Pull request merged**